launchable/test_runners/launchable.py

```diff
@@ -5,7 +5,6 @@
 
 import click
 
-from launchable.app import Application
```

Comment:
@Konboi your last change in this field left this dangling import statement. I wonder if your pre-commit is set up correctly? It should have caught this.

Reply:
Oh, really? Sorry, I'll check my pre-commit config.